src/searchForm/index.tsx

```diff
@@ -0,0 +1,165 @@
+  }
+
+  & .kirk-searchForm-from-container,
+  & .kirk-searchForm-dateSeat-container {
+    display: flex;
+  }
+
+  & .kirk-searchForm-textfield .kirk-textField-wrapper {
+    height: auto;
+    background: none;
+    border: none;
+  }
+
+  & .kirk-searchForm-textfield input {
+    background: none;
+  }
+
+  & .kirk-searchForm-button {
+    color: ${color.secondaryText};
+  }
+
+  & .kirk-bullet--searchForm {
+    margin-right: ${space.m};
+  }
+
+  & .kirk-searchForm-invert {
+    padding-right: ${space.m};
+    border-right: 1px solid ${color.inputBorder};
+  }
+
+  & .kirk-searchForm-invert .kirk-button {
+    display: flex;
+  }
+
+  & .kirk-searchForm-invert .kirk-icon {
+    transform: rotate(90deg);
+  }
+
+  & .kirk-searchForm-from .kirk-button,
+  & .kirk-searchForm-to .kirk-button,
+  & .kirk-searchForm-date .kirk-button,
+  & .kirk-searchForm-seats .kirk-button {
+    display: flex;
+  }
+
+  & .kirk-searchForm-from .kirk-button,
+  & .kirk-searchForm-to .kirk-button {
+    width: ${primaryFieldsWidth};
+    border-right: 1px solid ${color.inputBorder};
+    padding-left: ${space.m};
+  }
+
+  & .kirk-searchForm-date .kirk-button,
+  & .kirk-searchForm-seats .kirk-button {
+    width: ${secondaryFieldsWidth};
+    padding-left: ${space.m};
+  }
+
+  & .kirk-searchForm-from .kirk-button {
+    border-right: none;
+  }
+
+  & .kirk-searchForm-date .kirk-button {
+    border-right: 1px solid ${color.inputBorder};
+  }
+
+  & .kirk-searchForm-submit .kirk-button {
+    width: ${submitButtonWidth};
+    height: ${formHeight};
+    border-radius: 0 ${radius.l} ${radius.l} 0;
+  }
+
+  & .kirk-searchForm-submit .kirk-icon {
+    margin: 0;
+  }
+
+  & .kirk-button svg {
+    margin: 0;
+    margin-right: ${space.m};
+  }
+
+  @media (${responsiveBreakpoints.isMediaSmall}) {
+    & {
+      flex-direction: column;
+      width: auto;
+      height: auto;
+      padding: ${space.xl};
+      padding-bottom: 0;
+      align-items: stretch;
+    }
+
+    & .kirk-searchForm-from .kirk-button,
+    & .kirk-searchForm-to .kirk-button,
+    & .kirk-searchForm-date .kirk-button,
+    & .kirk-searchForm-seats .kirk-button {
+      padding: 0;
+      width: 100%;
+    }
+
+    & .kirk-searchForm-from {
+      flex-grow: 1;
+    }
+
+    & .kirk-searchForm-invert {
+      padding: 0;
+      border: none;
+    }
+
+    & .kirk-searchForm-to .kirk-button {
+      border: none;
+    }
+
+    & .kirk-searchForm-date,
+    & .kirk-searchForm-seats {
+      flex: 1;
+    }
+
+    & .kirk-searchForm-to .kirk-button {
+      width: 100%;
+    }
+
+    & .kirk-searchForm-dateSeat-container {
+      padding-bottom: ${space.m};
+    }
+
+    & .kirk-searchForm-submit {
+      margin-left: -${space.xl};
+      margin-right: -${space.xl};
+    }
+
+    & .kirk-searchForm-submit .kirk-button {
+      width: 100%;
+      border-radius: 0 0 ${radius.l} ${radius.l};
+      text-align: center;
+    }
+
+    & .kirk-searchForm-submit .kirk-icon {
+      flex: 1;
```

Comment:
why do we need that ?

Reply:
It's centering the icon, there might be a better way.